model/pdata/metrics.go

```diff
@@ -247,6 +247,22 @@ func (at AggregationTemporality) String() string {
 	return otlpmetrics.AggregationTemporality(at).String()
 }
 
+// DataPointFlags defines how a metric aggregator reports aggregated values.
+// It describes how those values relate to the time interval over which they are aggregated.
+type DataPointFlags uint32
```

Comment:
s/DataPointFlags/MetricDataPointFlags?

Reply:
Not sure but this will make it consistent with MetricDataType, probably we should have "MetricAggregationTemporality" as well.